### ISSUE TITLE
Freemode Male & Female missing

### ISSUE THREAD

**Issue by ThatRealSpicyDragon** [itsjustcurtis/MenyooSP#68]
Freemode Male and Female (the Online protag models) are missing from ped spawner and player models.

**Comment by ThatRealSpicyDragon** [comment]
Update: I also did not see a Player category.

**Comment by ThatRealSpicyDragon** [comment]
The "Player" category seems to be missing in both menus too. 
![Grand Theft Auto V Screenshot 2024 02 25 - 16 52 21 60](https://github.com/itsjustcurtis/MenyooSP/assets/158034338/3da3de5a-c0c9-4955-90f5-24ee1e69d1da)
Just to prove 
![Grand Theft Auto V Screenshot 2024 02 25 - 16 52 41 22](https://github.com/itsjustcurtis/MenyooSP/assets/158034338/af441c4a-962d-4cad-b321-698ff04512b8)
I'm not blind as a bat.

**Comment by itsjustcurtis** [comment]
It looks like your pedlist.xml might be missing that section. Try using the one available in the repo [here ](https://github.com/itsjustcurtis/MenyooSP/blob/main/Solution/source/_Build/bin/Release/menyooStuff/PedList.xml)

**Comment by TayMcKenzieNZ** [comment]
Yeah I think that's my bad. I was trying to move Michael Trevor Franklin and MP Peds into the player category. Huge apologies!

**Comment by ThatRealSpicyDragon** [comment]
> It looks like your pedlist.xml might be missing that section. Try using the one available in the repo [here ](https://github.com/itsjustcurtis/MenyooSP/blob/main/Solution/source/_Build/bin/Release/menyooStuff/PedList.xml)

I must unfortunately report (and believe me, this dumbfounds even myself) that replacing the PedList.xml with the provided instance does not change anything.
Unless the visible PedList.xml in _menyoostuff_ is not the actual PedList.xml that Menyoo uses for its lists, then something appears to be dreadfully awry.

**Comment by TayMcKenzieNZ** [comment]
Yeah, it's extremely strange. When I updated the list, I thought to myself,
let's take out the mp peds and Michael Trevor Franklin and put them all
into one list so they're easily accessible, however like yourself, I also
noticed that player category is completely non existent and I have no idea
why. But to add to this, I haven't seen it in previous versions either so
it's not just you. 

I'm baffled because I don't see any syntax errors either

**Comment by TayMcKenzieNZ** [comment]
Apologies @itsjustcurtis @itsjustcurtis 😣

**Comment by chubbyc9** [comment]
is there any update for a fix on this ? im new to modding the game and i installed menyoo properly but in the "multiplayer models" there isnt a "Freedom female" option or a male option either, please help thank you.

**Comment by TayMcKenzieNZ** [comment]
You can open the pedlist.xml file and add them back into the Multiplayer section

```xml
		<Ped name="mp_f_freemode_01" caption="Freemode MP Female" />
		<Ped name="mp_m_freemode_01" caption="Freemode MP Male" />
```

**Comment by TayMcKenzieNZ** [comment]
Any plans to officially fix this, @itsjustcurtis ?

**Comment by itsjustcurtis** [comment]
> Any plans to officially fix this, @itsjustcurtis ?

I've got the fix locally, just not enough work in the patch branch right now to justify a release yet

**Comment by ThatRealSpicyDragon** [comment]
Will the missing decals and blips be added in the next branch? I tried to do it myself, but couldn't figure out how to compile the source code...

**Comment by itsjustcurtis** [comment]
> Will the missing decals and blips be added in the next branch? I tried to do it myself, but couldn't figure out how to compile the source code...

I'm not sure how that relates to this issue. Please raise as a separate issue if it hasn't been already.

**Comment by itsjustcurtis** [comment]
Resolved with 2.0.3
